<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>РПС. Лабораторная работа №3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="random_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>420</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>СГЕНЕРИРОВАТЬ СЛУЧАЙНЫЕ МАССИВЫ</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="output_table">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>930</width>
      <height>331</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>232</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>id</string>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>array</string>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>change_date</string>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>type</string>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="load_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>241</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>470</y>
      <width>241</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>931</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>СОРТИРОВАТЬ МАССИВЫ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Таблица вывода массивов</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_of_arrays">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>450</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Количество</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Не удалять данные в базе при сохранении массивов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Не удалять данные в таблице при загрузке массивов</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="save_in_db">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="save_in_table">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>random_btn</zorder>
   <zorder>load_btn</zorder>
   <zorder>save_btn</zorder>
   <zorder>sort_btn</zorder>
   <zorder>label</zorder>
   <zorder>output_table</zorder>
   <zorder>number_of_arrays</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>save_in_db</zorder>
   <zorder>save_in_table</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
